src/mono/wasm/debugger/BrowserDebugProxy/JObjectValueCreator.cs

```diff
@@ -265,22 +265,51 @@ async Task<string> GetNullObjectClassName()
             return className;
         }
     }
-
+    private async Task<string> ToString(List<int> typeIds, int objectId, CancellationToken token)
+    {
+        string description = "";
+        try {
+            foreach (var typeId in typeIds)
+            {
+                var typeInfo = await _sdbAgent.GetTypeInfo(typeId, token);
+                if (typeInfo != null && typeInfo.Name != "object")
```

Comment:
nit:
```suggestion
                if (typeInfo == null || typeInfo.Name == "object")
                    continue;
```

Reply:
When could `typeInfo` be `null`? If it is null for a valid type, and that has a `ToString` override, then we will be skipping that, and using a parent's `ToString`.

And that would be a useful test to add - derived, and parent class both having `ToString` overridden, just to confirm that we are picking the correct one.